oleggerasinovich/Titanic.ipynb

```diff
@@ -0,0 +1,1686 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    age_proportions.reset_index(level=0, inplace=True)
</pre><p>Here you can leave the frame as it is, the visualization will take the index as X.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/u2-de-students-lab/titanic_exploration/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>